### ISSUE TITLE
 cannot import name 'Deconvolution3D' from 'keras.layers' (/opt/conda/lib/python3.7/site-packages/keras/layers/__init__.py)

### ISSUE THREAD

**Issue by sneh-debug** [ellisdg/3DUnetCNN#244]
from the unet.py script , while executing " from keras.layers import Conv3D, MaxPooling3D, UpSampling3D, Activation, BatchNormalization, PReLU, Deconvolution3D", i got following error:
cannot import Deconvolution3D from keras.layers.
I have tensorflow-gpu==2.2.0 and keras=2.3.0-tf.

**Comment by Chianti1114** [comment]
Hi, please try Conv3DTranspose:
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv3DTranspose
